### ISSUE TITLE
Feature Request: Output Monitoring

### ISSUE THREAD

**Issue by polyh3dron** [Andersama/obs-asio#105]
Putting in an official feature request here to allow OBS to select ASIO outputs for monitoring, in both the main monitor output of OBS and in the Audio Monitor filter. As it currently stands, the use of ASIO inputs in OBS with this plugin effectively renders the device's outputs unusable.

**Comment by fbosman** [comment]
We need this also!

**Comment by parsivan** [comment]
Yeah i mean i don't know how to monitor the setup without doing a test record

**Comment by DrummyFloyd** [comment]
i'm totally agree with that =D

**Comment by christophermh44** [comment]
Any news about this feature?
I think it would be great to have an audio filter that route a source audio to an ASIO output.

Use case: when using Twitch Guest Star, it will become possible to process the audio of each guest displayed in the stream with a DAW (like Reaper) and send back the processed audio to OBS.

**Comment by maduranma** [comment]
Yes please! This would be a game changer!

**Comment by polyh3dron** [comment]
I suppose this isn't going to happen?

**Comment by jasonseeb** [comment]
> I suppose this isn't going to happen?

The author of the plugin already said they would not add this as stated in the first issue post right before this one in May of 2021. They list the reasons and workarounds for it in the same post.

**Comment by polyh3dron** [comment]
> > I suppose this isn't going to happen?
> 
> The author of the plugin already said they would not add this as stated in the first issue post right before this one in May of 2021. They list the reasons and workarounds for it in the same post.

As I said in the initial post of this thread, this is not a request for output capture.

**Comment by Andersama** [comment]
@jasonseeb @polyh3dron I know what polyh3dron's after, I never bothered to do output monitoring because my setup at the time it would've been redundant. However to jason's point, I haven't touched anything obs related for years now and I don't plan to. @pkviet told me he rewrote the plugin from scratch a while ago removing its dependency on Juce, but since I remember the pita asio drivers and things were I didn't want him to swap that in the latest obs version bump. I feel like he might've added this in.

I'm still not in favor of dropping the Juce library since that was the stablest we had the plugin. If he sees this, and my comment here maybe he'll work it in this version.

**Comment by PeGeve** [comment]
where can i get his versions because i'd appreciate if i could get it with output.
